### ISSUE TITLE
Robots should have flange

### ISSUE THREAD

**Issue by Doomerdinger** [ros-industrial/motoman#374]
Each robot should have flange, and tool0 should be a child of flange not the last robot link.
See [REP-0199](http://gavanderhoorn.github.io/rep/rep-0199.html).

Related: #21

**Comment by gavanderhoorn** [comment]
I agree with this, but note: the REP you mention is not an official REP.

So we can decide we want to adhere to it here, but we can't "fault" the current models for not having those frames.

**Comment by Doomerdinger** [comment]
Yup, I'm already throwing something together for it and since things like FANUC already do it (thanks to you I believe) I figured I'd open something up.
